File: app/src/main/java/id/ac/unej/inaparadise/ui/LoginActivity.java
package id.ac.unej.inaparadise.ui;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

import id.ac.unej.inaparadise.MainActivity;
import id.ac.unej.inaparadise.R;

public class LoginActivity extends AppCompatActivity {

    EditText email, password;
    Button daftar, masuk;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);
        email = findViewById(R.id.email);
        password = findViewById(R.id.password);

        daftar = findViewById(R.id.daftar);
        masuk = findViewById(R.id.masuk);

        daftar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(LoginActivity.this, RegisterActivity.class));
            }
        });

        masuk.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(LoginActivity.this, MainActivity.class));
            }
        });
    }
}

File: app/src/main/java/id/ac/unej/inaparadise/ui/LoginActivity.kt
package id.ac.unej.inaparadise.ui

import android.widget.Button
import id.ac.unej.inaparadise.MainActivity

class LoginActivity : AppCompatActivity() {
    var email: EditText? = null
    var password: EditText? = null
    var daftar: Button? = null
    var masuk: Button? = null
    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        email = findViewById<EditText>(R.id.email)
        password = findViewById<EditText>(R.id.password)
        daftar = findViewById<Button>(R.id.daftar)
        masuk = findViewById<Button>(R.id.masuk)
        daftar!!.setOnClickListener {
            startActivity(
                Intent(
                    this@LoginActivity,
                    RegisterActivity::class.java
                )
            )
        }
        masuk!!.setOnClickListener {
            startActivity(
                Intent(
                    this@LoginActivity,
                    MainActivity::class.java
                )
            )
        }
    }
}